### ISSUE TITLE
vmd crashes before start

### ISSUE THREAD

**Issue by caske33** [yoshuawuyts/vmd#86]
I just updated vmd to version 1.25.1 and now it won't start anymore :cry: 

```
App threw an error when running [TypeError: Cannot read property 'on' of undefined]
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError: Cannot read property 'on' of undefined
    at Object.<anonymous> (/usr/lib/node_modules/vmd/main/main.js:65:4)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at loadApplicationPackage (/usr/lib/node_modules/vmd/node_modules/electron-prebuilt/dist/resources/default_app/main.js:257:23)
    at Object.<anonymous> (/usr/lib/node_modules/vmd/node_modules/electron-prebuilt/dist/resources/default_app/main.js:289:5)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
```

I really love this app, so is there anything I can do to fix it?

**Comment by maxkueng** [comment]
Pretty weird. I can't reproduce the error. But if it `Cannot read property 'on' of undefined` at `main/main.js:65` then something most be wrong with Electron because that's where it `app.on('window-all-closed', ...` and `app` should never be undefined.

Anyway, I've upgraded Electron to version 1.2.1 and published vmd@1.26.0. Could you just install the new version and close this issue if the error is gone?

```
npm install -g vmd@1.26.0
```

**Comment by caske33** [comment]
Indeed a problem with electron, I guess. It's gone after the update. Thx :+1:
